src/neo/Trie/MPT/MPTDatabase.cs

```diff
@@ -0,0 +1,50 @@
+using Neo.Persistence;
+using System.Text;
+
+namespace Neo.Trie.MPT
+{
+    public class MPTDatabase
+    {
+        private ISnapshot store;
+        public static readonly byte TABLE = 0x4D;
+        public static readonly byte[] Prefix = Encoding.ASCII.GetBytes("MPT");
+        public static readonly byte[] ROOT_KEY = Encoding.ASCII.GetBytes("MPT_ROOT");
```

Comment:
Plz, unify the names.
```
TABLE => Table
Prefix => Prefix
ROOT_KEY => Root = Encoding.ASCII.GetBytes("ROOT") 
```
Do we need the prefix? I'm not sure, as we have prefix in contract/storage/block/xxx already.

Reply:
I think there is no all caps case in neo repository. plz rename them
```
TABLE => Table
ROOT_KEY => RootKey
```
And also the `CHILD_COUNT => MaxChildren`  in `FullNode`.